src/printer/tvmscript_printer.cc

```diff
@@ -577,6 +577,9 @@ bool TVMScriptPrinter::IsSimpleBuffer(const Buffer& buf) {
     return false;
   }
   for (const PrimExpr& shp_i : buf->shape) {
+    if (shp_i.dtype() != DataType::Int(32)) {
+      return false;
+    }
```

Comment:
Hi. It’s true that we can choose not to print in the sugar type annotation style when the shape contains `int64`, but what I’m curious is, can’t we enhance the type annotation to support `int64`? That way makes more sense, since we always want to make our stuff better and work in different cases.

Reply:
@MasterJH5574 The initial code change of this PR is to act as a temporary fix to address your [comment](https://github.com/apache/tvm/pull/10789#issuecomment-1079625981) here.

Reply:
The current code change is to change the definition of `ConcreteType` so it could take more than one positional arguments.